core/rfb.js

```diff
@@ -2025,6 +2052,48 @@ export default class RFB extends EventTargetMixin {
         return true;
     }
 
+    _handleReplitAudioPseudoEncodingMsg() {
+        if (this._sock.rQwait("Replit audio message", 3, 1)) { return false; }
+        const submessage = this._sock.rQshift8();
+        const length = this._sock.rQshift16();
+        if (this._sock.rQwait("Replit audio message", length, 4)) { return false; }
+
+        switch (submessage) {
+            case 0: { // CaptureState response.
+                const enabled = this._sock.rQshift8() == 1;
+
+                if (enabled) {
+                    this._audioStream = new AudioStream(this._audioMimeType);
+                    RFB.messages.ReplitAudioEnableContinuousUpdate(this._sock);
+                } else {
+                    this._audioStream.close();
+                }
+                break;
+            }
+
+            case 1: { // AudioFrame response.
+                const timestamp = this._sock.rQshift32();
```

Comment:
Argh, forgot to add the MSB check for the keyframes so that the AudioBuffer can retire old chunks and not show a warning on Firefox. I'll add it later.